### ISSUE TITLE
Any game stops randomly when Sync Video is on

### ISSUE THREAD

**Issue by themalu866** [mgba-emu/mgba#1336]
Playing on Windows, when Sync Video is ON the games just randomly stops but doesn't freezes the emulator.
I say this because mGBA still works, I can do quick saves and if I go to settings and disable sync video the games go back to normal, with the exception when I tried to record a gif and then disable it, it didn't work.
I can't give save states when it stops because if I close the emulator and then open and load it the game works just fine.
This happens with any game, GBA GBC and GB and it doesn't matter if Pause when inactive is ON or OFF.

![trew](https://user-images.githubusercontent.com/42208976/53695380-c8d1e400-3d99-11e9-8027-6eed5a79c973.gif)

**Comment by endrift** [comment]
Is sync to audio still enabled when you do this?

**Comment by themalu866** [comment]
Yes, because if audio sync id off then it starts doing crackling sounds and it's really annoying.

**Comment by WloHu** [comment]
I have the same issue:
- Both on -> random stops (mgba-qt UI is still responsive),
- Both off -> fast forward maxes out w/o keypress,
- A-on, V-off -> playable best setting,
- A-off, V-on -> audio crackles.
